tracing/tracer.py

```diff
@@ -164,30 +164,35 @@ def _on_trace_is_called(self, frame, event, arg: typing.Any) -> typing.Callable:
         line_number = frame.f_lineno
 
         names2types, category = None, None
+        frameinfo = inspect.getframeinfo(frame)
+
         if event == "call":
-            logger.info(f"Tracing call: {inspect.getframeinfo(frame)}")
+            logger.info(f"Tracing call: {frameinfo}")
             names2types = self._on_call(frame, arg)
             category = TraceDataCategory.FUNCTION_ARGUMENT
 
         elif event == "return":
-            logger.info(f"Tracing return: {inspect.getframeinfo(frame)}")
+            logger.info(f"Tracing return: {frameinfo}")
             names2types = self._on_return(frame, arg)
             category = TraceDataCategory.FUNCTION_RETURN
 
+            # Special case
+            line_number = 0
+
             # Adds tracing data of class members if the return is from a class function.
             if possible_class is not None:
                 names2types2 = self._on_class_function_return(frame)
                 category2 = TraceDataCategory.CLASS_MEMBER
-                self._update_trace_data_with(file_name, possible_class, "", 0, category2, names2types2)
+                self._update_trace_data_with(file_name, possible_class, "", line_number, category2, names2types2)
```

Comment:
The former case with 0 is preferred as the line number variable name does not indicate that it is always 0.

Reply:
Fixed